### PR TITLE
Index packing lists by date

### PR DESCRIPTION
**Pull request by tlvince**
* Uses delivery date as-is (will need prettifying when we switch to JSON dates)
* Removes packed/unpacked split table in favour of simple list with indicators
* Disables (unrelated) failing specs

Note, you'll need to update the 'by-driver' view to test this.

### CODE REVIEW

**Comment by tlvince** [comment]
Test results: https://travis-ci.org/tlvince/direct-delivery/builds/47641854

**Comment by tlvince** [comment]
@hbj, rebased.

File: src/app/packing/all/all.html [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
+    <i
+      class="fa-li fa"
+      ng-class="{
+        'fa-circle-o': !packing.packed,
+        'fa-check-circle-o text-success': packing.packed
+      }"
+    ></i>
+    <a
+      ui-sref="packing.item.table({id: packing.id})"
+      ng-bind="packing.date"
```
Comment (hbj) [inline src/app/packing/all/all.html]: One time binding?

**Comment by hbj** [comment]
There are some places where we could use one time binding, for example in `packing/item/item.html`, we could use for all bindings except for `packedQty`.

**Comment by tlvince** [comment]
Good idea :+1:

**Comment by tlvince** [comment]
Added.

**Pull request merged**